components/review-modal.tsx

```
@@ -0,0 +1,11 @@
+import React from 'react'
+
+type ReviewModalProps = {
+  closeModal: () => void
+}
+
+const ReviewModal: React.FC<ReviewModalProps> = ({ closeModal }) => {
```

ESLint

[error] 'closeModal' is defined but never used.

(@typescript-eslint/no-unused-vars)